cpp/src/traversal/bfs.cu

```diff
@@ -270,20 +271,20 @@ namespace cugraph {
 
               //We need to prepare the switch back to top down
               //We couldnt keep track of mu during bottom up - because we dont know what mf is. Computing mu here
-              count_unvisited_edges(unvisited_queue,
+	      bfs_kernels::count_unvisited_edges(unvisited_queue,
```

Comment:
@BradReesWork @afender

What is cuGraph's policy on namespaces?

If we're following cudf (https://github.com/rapidsai/cudf/blob/branch-0.8/cpp/docs/TRANSITIONGUIDE.md),

We should use cugraph:: for public API, cugraph::detail:: for cugraph internal implementations, and anonymous namespace for implementations restricted to a single translation unit.

Reply:
makes sense. let's add this to the c++ refactoring task.